client_test.go

```diff
@@ -1,19 +1,39 @@
 package iex
 
 import (
+	"io"
 	"net/http"
+	"net/http/httptest"
 	"testing"
 	"time"
 )
 
+type MockHTTPClient struct {
+	body    string
+	headers map[string]string
+	code    int
+	err     error
+}
+
+func (c *MockHTTPClient) Get(url string) (*http.Response, error) {
+	w := httptest.NewRecorder()
```

Comment:
Is this equivalent to `w.WriteString(c.body)`?

Reply:
Nearly enough to pass the tests, anyways 😄